src/serviceregistry/5_0_0/sysd/serviceregistry_sysd.tex

```diff
@@ -0,0 +1,228 @@
+
+The previous versions of the Service Registry (4.6.x) are very similar to this version, however there are some key differences, even in conceptual level:
+
+\begin{itemize}
+    \item There was no data storage separation requirement: the Service Registry's data storage was interconnected to other systems' storage. In the current version, data storage separation is mandatory.
+    \item Every service definition contained exactly one operation. In the current version, a service definition can consist of several operations.
+    \item The optional device information storage (and system information storage as well, to an extent) was handled by different systems. In the current version, the Service Registry handles all three level (service instances, systems, devices).
+    \item Service access information (e.g. domain name or IP address) was tied to the system that provides that service, and only one such address could be specified. In the current version, service access information is tied to the interface of a service instance, and multiple addresses can be provided.
+    \item Service interface representation was very rudimentary, a much sophisticated approach is used in the current version.
+    \item X.509 certificate trust chains was used as authentication mechanism. The current version can support any type of authentication methods by using a dedicate Authentication core system. However, a slightly modified version of X.509 trust chains are still supported.
+\end{itemize}
+
+\subsection{How This System Is Meant to Be Used}
+\label{sec:use}
+
+Service Registry is a mandatory core system of Eclipse Arrowhead Local Cloud and is responsible for the fundamental service discovery functionality. Systems and services (and devices) are being discoverable for other systems by consuming the services provided by Service Registry core system.
+
+An application or other core/support system is required to register itself and its services into Service Registry if it is designed to be part of the Local Cloud and to unregister before shutdown or when it does not intend to continue to provide its services.
+
+\subsection{System functionalities and properties}
+\label{sec:properties}
+
+\subsubsection {Functional properties of the system}
+Service Registry solves the following needs to fulfill the requirements of service/system/device discovery.
+
+\begin{itemize}
+    \item Enables the application and other core/support systems to register themselves in order to being part of the Local Cloud.
+    \item Enables the application and other core/support systems to unregister themselves from the Local Cloud.
+    \item Enables the application and other core/support system to register the device on which the system is running.
+    \item Enables the application and other core/support system to unregister the device on which the system is running (only if no other systems belongs to that device).
```

Comment:
"no other system**s** belongs to that device" -> "no other system belongs to that device"